README.md

```diff
@@ -1,5 +1,22 @@
 # chronologger
 
-To run, do `go run .`
+Chronologger is a personal review app, built with Golang.
 
-Please submit a PR first TEST
+# Development
+
+Start the application with hot module reloading using `make hmr`.
+
+Alternatively, if you don't need hot module reloading, you can just run `go run .` or `make start`
+
+## Known issues
+
+```
+(!) templ version check failed: failed to parse go.mod file: ~/chronologger/go.mod:3: invalid go version '1.21.0': must match format 1.23
+```
+To fix this, make sure you have at least the following version installed:
+```
+go install github.com/a-h/templ/cmd/templ@v0.2.543
+```
+You can test this by running `templ version`.
```

Comment:
I had to setup my `GOPATH` locally before I installed it. Not sure why, since go modules should stop this being needed.

Reply:
<img width="699" alt="Screenshot 2024-02-11 at 17 00 12" src="https://github.com/hay-i/chronologger/assets/21108297/ef35e96b-421d-4561-9d0e-bc23b76fdd18">
This is what I did per https://github.com/asdf-community/asdf-golang/issues/28 . Not sure if that's the preferred approach or not